backend/codesirius/codesirius/exception_handler.py

```
@@ -2,67 +2,56 @@
-                "errors": formatted_errors,
-            }
 
-        elif isinstance(exc, APIException):  # Handle generic API exceptions
-            response.data = {
-                "timestamp": timestamp,
-                "status": response.status_code,
-                "error": HTTPStatus(response.status_code).phrase,
-                "message": exc.detail if exc.detail else exc.default_detail,
-                "path": context["request"].path,
-            }
+def custom_exception_handler(exc, context):
```

Ruff (0.8.2)

Missing return type annotation for public function `custom_exception_handler`

(ANN201)